app/Models/Product.js

```
@@ -0,0 +1,32 @@
+'use strict'
+
+const Model = use('Model')
+const uuid = require('uuid/v4')
+
+class Product extends Model {
+	static boot() {
+		super.boot()
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)